src/core/AnimatedJSFunc.js

```diff
@@ -0,0 +1,17 @@
+import AnimatedNode from './AnimatedNode';
+import { adapt } from './AnimatedBlock';
+
+class AnimatedJSFunc extends AnimatedNode {
+
+  constructor(code, args) {
+    super({ 
+      type: 'jsfunc', 
+      args: args.map(n => n.__nodeID),
+      code: code,
+    }, [...args]);
+  }
```

Comment:
wouldn't this need to implement an `__onEvaluate` as well?

Reply:
In this draft (which will probably be deleted) only the native side was implemented to test if this was a viable way to get faster executing by reducing the number of nodes.